mod/metrics/set/abstract/researched.rb

```diff
@@ -1,115 +0,0 @@
-def score_cards
```

Comment:
this was a duplicate of `metrics/set/abstract/01_researched.rb` with a few fixes.
`abstract/relationship` included `01_researched.rb` without the fixes and then `researched.rb`  overrode it. I moved all the changes to `01_researched.rb`